### ISSUE TITLE
tests in pypi tarball + unittest.mock

### ISSUE THREAD

**Issue by pgajdos** [dkarchmer/django-rest-framework-client#7]
Hi,

could you please consider to ship tests in release tarball? Also, you could eventually replace mock with built-in unittest.mock.

**Comment by dkarchmer** [comment]
Yes, I don't have too much of a problem with this (even if I am not sure I understand why you would need this), but not sure when I can do any work here. But happy to accept PRs if you want to help me with this @pgajdos

**Comment by pgajdos** [comment]
`import mock` does not seem to be needed at all actually. We need tests to check the functionality during package builds.

**Comment by pgajdos** [comment]
Feel free to close this in case you do not want to include tests.

**Comment by pgajdos** [comment]
(Or consider to use tags, like e. g. https://github.com/jazzband/djangorestframework-simplejwt does.)
